reps/2024-04-24-ray-obs-support.md

```diff
@@ -0,0 +1,103 @@
+## Summary
+
+### General Motivation
+
+Ray users can use the submit_job Api to send jobs to the ray cluster.
+
+At the same time, they can specify the location of the project's working directory in a remote file system, for example s3.
+
+This document suggests expanding the list of supported file systems and adding support for OBS (Object Storage Service).
```

Comment:
I assume this is https://www.huaweicloud.com/intl/en-us/product/obs.html ? Would be good to add a reference to clarify

Reply:
Done, added link to docs.